target/postfix/main.cf

```diff
@@ -71,7 +71,6 @@ postscreen_greet_action = enforce
 postscreen_bare_newline_action = enforce
 
 # SASL
-smtpd_sasl_auth_enable = yes
```

Comment:
We could think about just replacing the `yes` with `no`; this may make it easier to change?